dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -377,6 +377,11 @@
                       timeUnits="minutes"
                       path="/core-service=platform-mbean/type=memory"
                       attribute="heap-memory-usage#committed" />
+          <metric-dmr name="Heap Max"
```

Comment:
I only named it that to be consistent with the names of the other similar metrics:
   "Heap Used"
   "Heap Committed"